lib/fathead/apple_docs_ios/parse.py

```diff
@@ -0,0 +1,144 @@
+# Introduction:
+# -------------
+# This is used to open an SQLite database that's installed on your computer to create a Fathead output
+# that's specified in http://docs.duckduckhack.com/resources/fathead-overview.html
+#
+# Page Structure:
+# ---------------
+# There is no page structure here because we're essentially scraping a database. The only requirement is that you have XCode.
+#
+# Pipeline:
+# ---------
+# Read DB -> Scrape / Process -> Output
+#
+# How do I test this?
+# ------------------
+# You'll need to have XCode installed on your computer. It comes bundled with the documentation that we need.
+#
+# TODO:
+# - Get disambiguations working
+#
+# Updates:
+#
+# August 30, 2016
+# - Now works with DocSets included in XCode 7.3.1
+
+import sqlite3
+import re
+import sys
+
+reload(sys)
+sys.setdefaultencoding('utf8')
+
+# These contains all of the API documentation
+# It only has classes and methods--it doesn't have actual tutorials.
+ios = "/Applications/Xcode.app/Contents/Developer/Documentation/DocSets/com.apple.adc.documentation.iOS.docset/Contents/Resources/docSet.dsidx"
```

Comment:
`download/com.apple.adc.documentation.iOS.docset/Contents/Resources/docSet.dsidx` -- the script has to be run on our Linux production machines so this path is never going to exist.

We'll need a `fetch.sh` script that gets the docset from somewhere and puts it into a local `download/` folder